.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Crate
```

Comment:
Since both of the large templates listing all of the services are generated using templates, I don't think there is a benefit to using the reusable actions. I was using them and removed them.